mentors/python/n.json

```diff
@@ -70,5 +70,13 @@
     "link_url": "https://nahua.dev",
     "avatar_url": null,
     "bio": "I am a self-taught programmer with a background in liberal arts. I work with Python professionally in the intersection between software development and deep learning. In my spare time, I also play with Go, Rust, and Swift."
+  },
+  {
+    "github_username": "nathalier",
+    "name": "Natalie R",
+    "link_text": null,
+    "link_url": " ",
```

Comment:
URL cannot be white space.  Should be `null` if you do not have a link you wish to share.